kotlin-data-access-provider-archetype/src/main/resources/archetype-resources/src/main/kotlin/__pluginName__/__pluginName__CredentialDataAccessProvider.kt

```diff
@@ -0,0 +1,33 @@
+package ${package}.${pluginName}
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import se.curity.identityserver.sdk.attribute.AccountAttributes
+import se.curity.identityserver.sdk.attribute.AuthenticationAttributes
+import se.curity.identityserver.sdk.datasource.CredentialDataAccessProvider
+
+class ${pluginName}CredentialDataAccessProvider(private val _configuration: ${pluginName}DataAccessProviderConfig): CredentialDataAccessProvider
+{
+    override fun updatePassword(accountAttributes: AccountAttributes )
```

Comment:
I noticed some misplaced whitespaces here and there... could be a good idea to auto-format the code.

Reply:
It's a problem with the archetype template files that autoformatting makes a total mess because of the placeholders.

Reply:
I tried to fix some odd spaces and indentations.